tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+import torch
+
+
+# Check if MPS is available and functional (for GitHub Actions)
+def is_mps_available_and_functional():
```

Comment:
Should this be in `tests.__init__.py` or maybe in `tests.utils.py` or even `cheetah.utils`?

Reply:
I moved it to `cheetah.utils` now